test/common.py

```diff
@@ -1875,29 +1875,27 @@ class BrowserCore(RunnerCore):
   def __init__(self, *args, **kwargs):
     super().__init__(*args, **kwargs)
 
-  @staticmethod
-  def browser_open(url):
-    if not EMTEST_BROWSER:
-      logger.info('Using default system browser')
-      webbrowser.open_new(url)
-      return
+  @classmethod
+  def browser_restart(cls):
+    # Kill existing browser
+    logger.info("Restarting browser process")
+    cls.browser_proc.terminate()
+    try:
+      cls.browser_proc.wait(1)
+    except subprocess.TimeoutExpired:
+      cls.browser_proc.kill()
+      cls.browser_proc.wait()
+    cls.browser_open(cls.harness_url)
 
+  @classmethod
+  def browser_open(cls, url):
+    global EMTEST_BROWSER
+    if not EMTEST_BROWSER:
+      logger.info('No EMTEST_BROWSER set. Defaulting to `chrome`')
+      EMTEST_BROWSER = 'chrome'
```

Comment:
Shouldn't this be `google-chrome`? I think that's how I've always seen it packaged.

Reply:
Done